seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java

```diff
@@ -143,15 +143,7 @@ public void testSubmitJob() {
                                 .getNodeExtension()
                                 .createExtensionServices()
                                 .get(Constant.SEATUNNEL_SERVICE_NAME);
-        Awaitility.await()
-                .atMost(2, TimeUnit.MINUTES)
-                .untilAsserted(
-                        () ->
-                                Assertions.assertEquals(
-                                        JobStatus.RUNNING,
-                                        seaTunnelServer
-                                                .getCoordinatorService()
-                                                .getJobStatus(Long.parseLong(jobId))));
```

Comment:
Why removed this?

Reply:
> Why removed this?

I tested it on my machine and found that it gets stuck here because the task has already been completed.
The status of the job will be tested below, so I think this place can be removed.